lib/modules/platform/github/index.ts

```diff
@@ -719,7 +719,10 @@ export async function findPr({
       return false;
     }
 
-    if (!config.forkToken && config.repository !== p.sourceRepo) {
+    if (
+      !config.forkToken &&
+      config.repository?.toLowerCase() !== p.sourceRepo?.toLowerCase()
```

Comment:
```suggestion
      config.repository?.localeCompare(p.sourceRepo, undefined, { sensitivity: 'accent' }) === 0
```
better use `localeCompare`

- https://stackoverflow.com/a/2140723/10109857